docs/API-functions.md

```diff
@@ -1191,10 +1199,11 @@ enum SRT_REJECT_REASON srt_getrejectreason(SRTSOCKET sock);
 ```
 
 This function shall be called after a connecting function (such as `srt_connect`)
-has returned an error, which's code was `SRT_ECONNREJ`. It allows to get a more
-detailed rejection reason. This function returns a numeric code, which can be
-translated into a message by `srt_rejectreason_str`. The following codes are
-currently reported:
+has returned an error, which's code was `SRT_ECONNREJ`, or - if the socket used
+for connection has been set `SRTO_RCVSYN` - when the `SRT_EPOLL_ERR` event is set
+for this socket. It allows to get a more detailed rejection reason. This
+function returns a numeric code, which can be translated into a message by
+`srt_rejectreason_str`. The following codes are currently reported:
```

Comment:
```suggestion
has returned an error, the code for which is `SRT_ECONNREJ`. If `SRTO_RCVSYN` has been set on the socket used for the connection, the function shall also be called when the `SRT_EPOLL_ERR` event is set
for this socket. This function provides a more detailed reason for the rejection. It returns a numeric code, which can be translated into a message by `srt_rejectreason_str`. The following codes are currently reported:
```

Reply:
The original sentence was too long. I tried to break it into two, but may have misinterpreted the substance. Please keep this in mind as you review my suggestions.

Reply:
I modified this a little bit:
1. Changed `shall` into `should` - calling this function isn't obligatory, but just useful.
2. Moved the explanation what the function does, which was in the middle, to the very beginning.
3. Changed "rejection" into "failed connection attempt" - as the reporting now includes also the case of connection timeout, so it can be used not only in case of rejected connection, but in all cases of a failure from `srt_connect` that can be recognized in async mode by checking the epoll flag. For example, it is possible also to call it in the blocking mode if the returned error was `SRT_ENOSERVER` (connection timeout), although it will simply always return `SRT_REJ_TIMEOUT` in this case, so it wouldn't make any sense.